src/Factories/PDOFactory.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
```

Comment:
Thinking about this from a curriculum perspective, how about we delete this. Remove it as a dependency from the modal and the DIC, then have the students set this up as an in class exercise. give them practice making a factory.

we still have 2 other factories we can show them to demo.